src/commands/react-native/README.md

```diff
@@ -0,0 +1,120 @@
+
+```bash
+# Environment setup
+export DATADOG_API_KEY="<API KEY>"
+```
+
+It is possible to configure the tool to use Datadog EU by defining the `DATADOG_SITE` environment variable to `datadoghq.eu`. By defaut the requests are sent to Datadog US.
+
+It is also possible to override the full URL for the intake endpoint by defining the `DATADOG_SOURCEMAP_INTAKE_URL` environment variable.
+
+## Commands
+
+### `upload`
+
+This command will upload all javascript sourcemaps and their corresponding bundle file to Datadog in order to un-minify front-end stack traces received by Datadog.
+
+To upload the sourcemaps for iOS, this command should be run:
+
+```bash
+datadog-ci react-native upload --platform ios --service com.company.app --bundle ./main.jsbundle --sourcemap ./main.jsbundle.map --release-version 1.23.4
+```
+
+To upload the sourcemaps for android, this command should be run:
+
+```bash
+datadog-ci react-native upload --platform android --service com.company.app --bundle ./index.android.bundle --sourcemap ./index.android.bundle.map --release-version 1.23.4
+```
+
+* `--platform` (required) identifies whether you are uploading ios or android sourcemaps.
+
+* `--service` (required) should be set as the name of the service you're uploading sourcemaps for, and Datadog will use this service name to find the corresponding sourcemaps based on the `service` tag set on the RUM SDK.
+By default the React Native SDK uses your app's bundle identifier as service.
+
+* `--bundle` (required) should be set to the path to your generated js bundle file, `main.jsbundle` for iOS or `index.android.bundle` for Android.
+
+* `--sourcemap` (required) should be set to the path to your generated sourcemap file, `main.jsbundle.map` for iOS or `index.android.bundle.map` for Android.
+
+* `--release-version` (required) will be used to match the `version` tag set on the RUM SDK.
+This should be the "Version" or "MARKETING_VERSION" in XCode for iOS and the "versionName" in your `android/app/build.gradle` for Android.
+
+* `--build-version` (required) is used to avoid overwriting your sourcemaps by accident.
+You cannot upload new sourcemaps for a specific `build-version` and `service` combination.
+This should be the "Build" or "CURRENT_PROJECT_VERSION" in XCode for iOS and the "versionCode" in your `android/app/build.gradle` for Android.
+
+In addition, some optional parameters are available:
+
+* `--disable-git` (default: false): prevents the command from invoking git in the current working directory and sending repository related data to Datadog (hash, remote URL and the paths within the repository of the sources referenced in the sourcemap).
+* `--dry-run` (default: `false`): it will run the command without the final step of upload. All other checks are performed.
+* `--repository-url` (default: empty): overrides the repository remote with a custom URL. For example: https://github.com/my-company/my-project
+
+### Link errors with your source code
```

Comment:
That's true but I'm not sure where it would live. Explaining under each command's readme what it's collecting on the local repository does make sense.